package.json

```diff
@@ -175,7 +176,7 @@
     "react-number-format": "^3.6.2",
     "react-papaparse": "^3.18.2",
     "react-popper": "^0.10.4",
-    "react-redux": "7.0.2",
+    "react-redux": "^7.2.5",
```

Comment:
did you mean to upgrade the version

Reply:
No, but the latest version of anything is better if it doesnt bring anything.